File: preview/understand/style_sheet/update-example-1/update-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Preview.Understand.Assistant;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var styleSheet = new Dictionary<string, Object>()
        {
            {"voice", new Dictionary<string, Object>()
                {
                    {"say_voice", "Polly.Joanna"}
                }}
        };

        var styleSheet = StyleSheetResource.Update(
            styleSheet: styleSheet,
            pathAssistantSid: "UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(styleSheet.AssistantSid);
    }
}

File: preview/understand/style_sheet/update-example-1/update-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

style_sheet = @client.preview
                     .understand
                     .assistants('UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                     .style_sheet
                     .update(style_sheet: {
                          'voice' => {
                              'say_voice' => 'Polly.Joanna'
                          }
                      })

puts style_sheet.assistant_sid
